api/v2/models/bouquets/bouquets_model.py

```diff
@@ -0,0 +1,54 @@
+from sqlalchemy import (Column, String, Integer, Boolean, Enum)
+from sqlalchemy.schema import Sequence
+
+
+from api.v2.helpers.database import Base
+from api.v2.utilities.utility import Utility, StateType
+
+
+class Bouquets(Base, Utility):
+    __tablename__ = 'bouquets'
+    id = Column(Integer, Sequence('bouquet_id_seq', start=1, increment=1), primary_key=True) # noqa
+    api_key1 = Column(String, nullable=False)
+    api_key2 = Column(String, nullable=False)
+    auth_credentials = Column(String, nullable=False)
+    bouquet_name = Column(String)
+    should_refresh = Column(Boolean, default=False)
+    state = Column(Enum(StateType), nullable=False, default="active")
+
```

Comment:
@koitoror are we missing the `refresh_url` field somewhere?

Reply:
Added. Thanks for noting.